src/http.ts

```diff
@@ -129,7 +132,11 @@ class Http {
       }
 
       xhr.onerror = function() {
-        reject(new RadarServerError());
+        if (host && host.includes('localhost')) {
```

Comment:
We should include the port here, it's possible that localhost is valid (ie. developing against dashboard locally)